contracts/bonding/BondingManager.sol

```diff
@@ -116,7 +124,8 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
      * @dev BondingManager constructor. Only invokes constructor of base Manager contract with provided Controller address
      * @param _controller Address of Controller that this contract will be registered with
      */
-    constructor(address _controller) public Manager(_controller) {}
+    constructor(address _controller) public Manager(_controller) {
```

Comment:
Let's revert this change since all other empty constructors have the brackets on the same line?

Reply:
fixed in 7fddb5f